crates/driver/src/domain/quote.rs

```diff
@@ -179,7 +179,7 @@ impl Order {
             // contract, so buy orders requiring excessively large sell amounts
             // would not work anyway.
             order::Side::Buy => eth::Asset {
-                amount: (eth::U256::one() << 192).into(),
+                amount: (eth::U256::one() << 144).into(),
```

Comment:
Is there a reason to use `144` specifically like there was to use `192`. If so it would be nice to link that here.

Reply:
Ah, sorry. This was the second time I read this PR but this time only focused on the code. 🤦‍♂️